.changeset/witty-seas-meow.md

```diff
@@ -0,0 +1,9 @@
+---
+'@aws-amplify/ui-vue': major
+---
+
+**Breaking**: We replaced following legacy Authenticator texts:
+- `Send Code` in reset password screen is replaced by `Send code`.
+- `Forgot your password? ` with the trailing slash is replaced by `Forgot your password`.
+
+If you were using `I18n` to translate those keys, please update your translations accordingly to match the new strings.
```

Comment:
I'll write migration guide docs in a separate PR, which will be almost verbatim as this changeset (Wanted to add on to migration guide that's already getting added in #2642). Any wording feedbacks appreciated!